### ISSUE TITLE
Can you upgrade to 1.20.4 and higher?

### ISSUE THREAD

**Issue by vaporvee** [AviatorRob/disx#1]
**Describe the bug**
It's not working on 1.20.4 

**Steps To Reproduce**
Use Neoforge 1.20.4 (I'm using a modpack)
And now nothing happens not even a crash

**Logs**
https://mclo.gs/OtGll7v

**Mod Loader**
Fabric, Forge, or Quilt?
- Forge / Neoforge

**Minecraft Version**
Pick from the one option available!
- 1.20.4

**Additional context**
Any additional information you feel necessary to share?

**Comment by aviatorrob06** [comment]
Hi @vaporvee,
The mod is still in beta development of some of its core features, so it currently is only being developed for version 1.20.1. As it progresses in development, I do plan on porting it to other versions, including 1.20.4. 

The primary reason that I have not started porting to other versions quite yet is because between Minecraft versions, its inner code often changes to a great extent. This means for each version I port to, I have to comb through the mod's code and modify it to function properly with these other versions of the game. If I were to port it now, I would have to develop updates for the mod for multiple copies. I am a full-time college student right now, so unfortunately my time to work on the mod is limited. For this reason, I will not port to other versions quite yet, but do keep your eyes peeled for when it does happen. 

I do apologize for the inconvenience of not having a build for your desired Minecraft version at this point in time.

**Comment by vaporvee** [comment]
Yeah have a similar issue with my mod i feel you. Maybe there is a solution for cross compatibillity, I'll let you know

**Comment by aviatorrob06** [comment]
> Yeah have a similar issue with my mod i feel you. Maybe there is a solution for cross compatibillity, I'll let you know

Sounds good.
